### PR TITLE
Restyle Bread Listing Page

### PR DESCRIPTION
**Pull request by jhancock532**
### [Link to GitPod Instance](https://gitpod.io/#https://github.com/jhancock532/bakerydemo/tree/restyle-bread-listing-page)

Updated the listing card component to display bread origin and type metadata, then populated the bread listing page with these new cards, updating the listing layout to use grid.

Desktop screenshot card borders look slightly off - this is a glitch with chrome devtools screenshot feature, I think a pixel or two gets missed occasionally.

### Screenshots
<details><summary>Expand to view more</summary>

8080-jhancock532-bakerydemo-vmwvwdz6bqh ws-eu44 gitpod io_breads_(iPhone SE)

8080-jhancock532-bakerydemo-vmwvwdz6bqh ws-eu44 gitpod io_breads_

<a href="#">Link to top</a>
</details>

### Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: bakerydemo/static/css/main.css [hunk 1]
```diff
@@ -673,8 +673,22 @@ footer {
...
+  text-transform: capitalize;
+}
+
+.pagination .active span {
+  background-color: var(--dark-orange);
+}
+
+.pagination .active span:hover {
+  background-color: var(--dark-orange);
 }
```
Comment (chris-lawton) [inline bakerydemo/static/css/main.css]:
```suggestion
.pagination .active span,
.pagination .active span:hover {
  background-color: var(--dark-orange);
}
```

File: bakerydemo/static/css/main.css [hunk 2]
```diff
@@ -1422,12 +1399,17 @@ input[type="radio"] {
...
 
 /* #region ---- Listing Card ---- */
-.blog-listing-card {
-  padding-bottom: 40px;
+.listing-card {
+  margin-bottom: 40px;
+}
+
+.listing-card__meta {
+  border-collapse:separate;
```
Comment (chris-lawton) [inline bakerydemo/static/css/main.css]:
```suggestion
  border-collapse: separate; 
```

File: bakerydemo/static/css/main.css [hunk 3]
```diff
@@ -1479,6 +1481,17 @@ input[type="radio"] {
...
+
+  .listing-card__meta-category {
+    font-size: 16px;
+    line-height: 24px;
+  }
+  
+  .listing-card__meta-content {
+    font-size: 16px;
+    line-height: 24px;
+  }
```
Comment (chris-lawton) [inline bakerydemo/static/css/main.css]:
```suggestion
  .listing-card__meta-category,
  .listing-card__meta-content  {
    font-size: 16px;
    line-height: 24px;
  }
```

File: bakerydemo/templates/breads/breads_index_page.html [hunk 4]
```diff
@@ -2,47 +2,21 @@
...
-                    </div>
-                </div>
-            </a>
-        {% endfor %}
-        </div>
+        <ul class="bread-list">
+            {% for bread in breads %}
+                {% include "includes/card/listing-card.html" with page=bread %}
+            {% endfor %}
+        </ul>
```
Comment (chris-lawton) [inline bakerydemo/templates/breads/breads_index_page.html:12]: so much neater!

File: bakerydemo/templates/includes/card/listing-card.html [hunk 5]
```diff
@@ -1,12 +1,28 @@
...
+    <a class="listing-card__link" href="{{ page.url }}">
+        {% if page.image %}
+            <figure class="listing-card__image">
+                {% image page.image fill-180x180-c100 %}
+            </figure>
+        {% endif %}
         <div class="listing-card__contents">
-            <h3 class="listing-card__title">{{ childpage.title }}</h3>
+            <h3 class="listing-card__title">{{ page.title }}</h3>
+            <table class="listing-card__meta">
```
Comment (chris-lawton) [inline bakerydemo/templates/includes/card/listing-card.html]:
```suggestion
{% if page.origin or page.bread_type %}
    <table class="listing-card__meta">
```

File: bakerydemo/templates/includes/card/listing-card.html [hunk 6]
```diff
@@ -1,12 +1,28 @@
...
+                {% if page.origin %}
+                    <tr>
+                        <td class="listing-card__meta-category">Origin</td>
+                        <td class="listing-card__meta-content">{{ page.origin }}</td>
+                    </tr>
+                {% endif %}
+                {% if page.bread_type %}
+                    <tr>
+                        <td class="listing-card__meta-category">Type</td>
+                        <td class="listing-card__meta-content">{{ page.bread_type }}</td>
```
Comment (chris-lawton) [inline bakerydemo/templates/includes/card/listing-card.html]: I combined the css rules that apply to these  classes in an earlier comment but I wonder if it's worth having 1 class for both instead?

**Comment by chris-lawton** [comment]
Thanks @jhancock532 - just a few comments/suggestions from me but looks good on the whole 👍

File: bakerydemo/templates/includes/card/listing-card.html (hunk 6 above)
Comment (jhancock532) [reply bakerydemo/templates/includes/card/listing-card.html]: The combination is the right idea for the desktop styles in the media query, however for mobile styling they have slightly different colours `--grey` and `--dark` so I think the class names should be kept seperate

Comment (chris-lawton) [reply bakerydemo/templates/includes/card/listing-card.html]: Ah my bad, I didn't stop the styles were different on mobile. Definitely worth keeping separate : )

**Pull request merged**